calico-enterprise_versioned_docs/version-3.16/variables.js

```diff
@@ -1,12 +1,12 @@
 const releases = require('./releases.json');
 
 const variables = {
-  releaseTitle: 'v3.16.0',
+  releaseTitle: 'v3.16.1',
   prodname: 'Calico Enterprise',
   prodnamedash: 'calico-enterprise',
   version: 'v3.16',
   baseUrl: '/calico-enterprise/3.16',
```

Comment:
@bmckercher123 The missing bit is this variable, which needs to be changed both here and in the 3.15 variables.js. Here, it needs to be changed to '/latest'. In 3.15, it needs to be changed from '/latest' to '/3.15'

(All of this happens automatically with the script, but you can also change it manually.

Reply:
I ran the script and the entries seem fine. However, the 3.15 version says "3.15.2" so if the script updated this, I assume it is correct.